testsuite/OMSysIdent/runCTests.py

```diff
@@ -1,8 +1,12 @@
 ## status: correct
-## teardown_command: rm test_HelloWorld_cs_Fit.log test_HelloWorld_cs_Fit_res.mat test_Lin2DimODE_cs_Fit.log test_Lin2DimODE_cs_Fit_res.mat
+## teardown_command: rm -rf test_HelloWorld_cs_Fit/ test_HelloWorld_cs_Fit.log test_HelloWorld_cs_Fit_res.mat test_Lin2DimODE_cs_Fit/ test_Lin2DimODE_cs_Fit.log test_Lin2DimODE_cs_Fit_res.mat
 
```

Comment:
This test should also state explicitly that it is only designed to run on linux:
```
## linux: yes
## mingw: no
## win: no
## mac: no
```